modules/shortcodes/spotify.php

```diff
@@ -75,7 +75,8 @@ function jetpack_spotify_embed_ids( $content ) {
 			continue;
 		}
 
-		if ( substr( ltrim( $element ), 0, 8 ) !== 'spotify:' ) {
+		// If this element does not contain a Spotify embed, continue.
+		if ( strpos( $element, 'spotify:' ) === false ) {
```

Comment:
What do you think about using yoda conditions here?

```suggestion
		if ( false === strpos( $element, 'spotify:' ) ) {
```

Reply:
Ahh. Yep. Let's do that instead.